_articles/ms/README.md

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
This file is probably causing many of the test failures. There shouldn't be a README file in translation folders. Delete it.

Reply:
alright, i'm going to remove it and then push again. Then see what errors there are and try to fix those.

Reply:
To be more accurate, I see that there is a README in zh-hant. It is OK to have a README, but the key thing is if it exists, it can't have frontmatter -- the beginning of the file surrounded by `---` lines, which causes it to be rendered as a page in the site.

Reply:
Alright, I've noticed that inside the en building-community file, that the "act immediately" link leads to a 404 page, is that intentional or an error that needs to be fixed? If it's intentional do I ignore the error that will also occur in the version I am currently translating to?

Reply:
It also occurs in the "ask them to leave", "explain why", "many ways to contribute", "A friendly README", "more than just a set of instructions", "your CONTRIBUTING file", link all lead to 404 pages.

Reply:
Or is it only an issue on my side? Sorry if I keep spamming issues left and right, I'm not familiar with GitHub and its many functionalities.

Reply:
That error is actually in the ms folder, see https://github.com/github/opensource.guide/pull/1737/checks?check_run_id=964363770#step:7:38

I'll make a comment at the line causing the problem.